content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -287,6 +287,32 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Deletion of kubectl on macOS
+
+The following methods exist for deleting ```kubectl``` on macOS:
+
+Type in the below command to check where kubectl is installed on your macOS:
+
+```bash
+where kubectl
+```
+
+(Usually the kubectl installation guides to move the ```kubectl``` binary to ```/usr/local/bin/``` , like ```/usr/local/bin/kubectl```)
+
+See the outputs of the command and type in the below command to remove it 
+
+```bash
+sudo rm <path> ##replace path with that of the output , eg : sudo sudo rm /usr/local/bin/kubectl
+```
+
```

Comment:
Replace <path> with the path to the kubectl binary from the previous step. For example, `sudo rm /usr/local/bin/kubectl`.

Reply:
I think `whereis  <argument>` the right command rather than `where`.

Reply:
And we can mention this command in one like - 

```bash
sudo rm $(whereis kubectl)
```

Reply:
```bash
whereis kubectl
```

seems to out more than one path for kubectl (ubuntu 22.04lts) - 
```
kubectl: /usr/bin/kubectl /usr/local/bin/kubectl
```

Instead we can mention - 

```bash
which kubectl
```

o/p -->> `/usr/local/bin/kubectl`

Reply:
where  works . Are you finding any issues with that @utkarsh-singh1 ?